requirements.txt

```diff
@@ -0,0 +1,4 @@
+ansible
```

Comment:
`python freeze > requirements.txt` will it not also provide version constraint on the dependencies so it's reproducible?

Reply:
The original is also not reproducible `pip3 install -U ansible pyyaml matplotlib signal-processing-algorithms`
It installs latest versions.
I don't know what was the original intent. But if you think it should be reproducible then I can add versions to each dependency.

Reply:
Sure, I understand. I think it might be worth doing and also adding something to the documentation to say something along the lines:

```bash
$ python3 -m venv venv
$ source venv/bin/activate
$ pip install -r requirements.txt
```

So to not pollute system install. As opposed to what you quoted -- I had meant to do this a while ago but forgot.

Reply:
Added versions to the requirement.txt